bindings/wasm/bindings.js

```diff
@@ -0,0 +1,99 @@
+var _ManifoldInitialized = false;
+Module.setup = function() {
+  if (_ManifoldInitialized) return;
+  _ManifoldInitialized = true;
+
+  function toVec(vec, list, f = x => x) {
+    for (let x of list) {
+      vec.push_back(f(x));
+    }
+    return vec;
+  }
+
+  function polygons2vec(polygons) {
+    return toVec(
+        new Module.Vector2_vec2(), polygons,
+        poly => toVec(new Module.Vector_vec2(), poly, p => {
+          if (p instanceof Array) return {x: p[0], y: p[1]};
+          return p;
+        }));
+  }
+
+  function vararg2vec(vec) {
+    if (vec[0] instanceof Array)
+      return {x: vec[0][0], y: vec[0][1], z: vec[0][2]};
+    if (typeof (vec[0]) == 'number')
+      // default to 0
+      return {x: vec[0] || 0, y: vec[1] || 0, z: vec[2] || 0};
+    return vec[0];
+  }
+
+  Module.Manifold.prototype.getGeometry = function() {
+    let mesh = this.getMesh();
+    let geometry = mesh2geometry(mesh);
```

Comment:
I think `getGeometry` should also be removed? Here's the downside of JS - no build errors.